deploy/helm/magda/charts/web-server/values.yaml

```diff
@@ -20,4 +20,4 @@ resources: {}
   #  memory: 128Mi
 disableAuthenticationFeatures: false
 service: {}
-showFallbackBanner: true
\ No newline at end of file
+fallbackUrl: "https://data.gov.au"
```

Comment:
We probably want to remove this line?
Asked because if the top-level config e.g. `minikube-dev.yml` doesn't have a `fallbackUrl:` line.
The default value from `deploy/helm/magda/charts/web-server/values.yaml` will be used.
So to turn redirection bar off, you will have to make sure the following in top-level config (e.g. `minikube-dev.yml`) :
```
web-server:
  fallbackUrl: ""
```